### PR TITLE
[loki] Add helm values schema

### PR DESCRIPTION
**Pull request by jkroepke**
#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #

#### Checklist

- [ ] [DCO](https://github.com/grafana-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [ ] Chart Version bumped
- [ ] Title of the PR starts with chart name (e.g. `[grafana]`)
- [ ] Add an [helm unittest](https://github.com/helm-unittest/helm-unittest) test case to cover this change.

### CODE REVIEW

File: charts/loki/tests/querier/keda_scaledobject_test.yaml [hunk 1]
```diff
@@ -123,17 +123,3 @@ tests:
       - equal:
           path: spec.pollingInterval
           value: 15
-
-  - it: should fallback to default pollingInterval when value is not an integer
```
Comment (QuentinBisson) [inline charts/loki/tests/querier/keda_scaledobject_test.yaml:127]: Why did you remove the keda tests?

Comment (jkroepke) [reply charts/loki/tests/querier/keda_scaledobject_test.yaml:127]: Because the json schema denies any non interger value.

Comment (QuentinBisson) [reply charts/loki/tests/querier/keda_scaledobject_test.yaml:127]: Then we probably don't need the safeInt helper anymore?

**Review by QuentinBisson — APPROVED** [review]

**Comment by jkroepke** [comment]
Whoever finds this: I expect some issues from now on due to the JSON schema changes. I tried to get high coverage with Helm unit tests, but sometimes I will not catch all edge cases.

Please report your issue as a GitHub issue instead of posting it here.

File: charts/loki/tests/querier/keda_scaledobject_test.yaml (hunk 1 above)
Comment (jkroepke) [reply charts/loki/tests/querier/keda_scaledobject_test.yaml:127]: The safeInt helper still resolves the following issue:

```
{{ 0 | default 1 }}
```

would resolve to `1` instead `0` .

But any guard rails to check, if the value is a number, can be omit.
